### ISSUE TITLE
Insert with returning value using prepared statement hanging

### ISSUE THREAD

**Issue by amsal** [sijms/go-ora#541]
When the following conditions are met, insert queries stop after first insert and wait indefinitely
- Insert query with a returning clause
- Prepared statement used for inserting more than 1 row
- varchar2 column values (arguments) are of different sizes, (other data types not tested, but number works as intended)

Database version is Oracle 19c and table (GOORA_TEMP_VISIT) ddl is from bulk insert example.

```go
package main

import (
	"database/sql"
	"fmt"
	"time"

	_ "github.com/sijms/go-ora/v2"
	goora "github.com/sijms/go-ora/v2"
)

func main() {
	urlOptions := map[string]string{
		"TRACE FILE": "trace.log",
	}
	connStr := goora.BuildUrl("localhost", 1521, "test.local", "sys", "password", urlOptions)
	conn, err := sql.Open("oracle", connStr)
	if err != nil {
		panic(err)
	}

	err = insertData(conn, false)
	if err != nil {
		fmt.Println("Can't insert data: ", err)
		panic(err)
	}

	err = insertData(conn, true)
	if err != nil {
		fmt.Println("Can't insert data: ", err)
		panic(err)
	}
}

func insertData(conn *sql.DB, variableLength bool) error {
	t := time.Now()
	index := 1
	stmt, err := conn.Prepare(`INSERT INTO GOORA_TEMP_VISIT(VISIT_ID, NAME, VAL, VISIT_DATE) 
VALUES(:1, :2, :3, :4) RETURNING VAL INTO :5`)
	if err != nil {
		return err
	}
	defer func() {
		_ = stmt.Close()
	}()
	nameText := "abcdefghi"
	val := 1
	start := 1
	end := 10
	if variableLength {
		nameText = "rstuvwxyz"
		start = 11
		end = 20
	}
	for index = start; index <= end; index++ {
		var out int
		if variableLength {
			nameText = fmt.Sprintf("%s%d", nameText, index)
		}
		_, err = stmt.Exec(index, nameText, val, time.Now(), goora.Out{
			Dest: &out,
			Size: 64,
			In:   false,
		})
		if err != nil {
			return err
		}
		val = val * 2
	}
	fmt.Println("10 rows inserted: ", time.Now().Sub(t))
	return nil
}
```

Running this program inserts the first 11 rows but then it hangs and no rows are inserted. I have narrowed the cause to variable length for the argument (nameText). For the 12th row, the size of the nameText is longer than 11th row's size.

**Comment by sijms** [comment]
fixed in next release

**Comment by sijms** [comment]
fixed v2.8.13
